common/components/controllers/AboutUserController.jsx

```diff
@@ -51,78 +53,97 @@ class AboutUserController extends React.PureComponent<{||}, State> {
-                <div className="col-12">
-                  <h2 className="text-uppercase">Technologies</h2>
-                  <div>{this._renderTechnologies()}</div>
-                </div>
-              </div>
-            ) : null}
-
-            <div className="row about-user-section">
-              <div className="col-12">
-                <h2 className="text-uppercase">About Me</h2>
-                <div style={{ whiteSpace: "pre-wrap" }}>
-                  {user && user.about_me}
-                </div>
-              </div>
+  _renderLeftColumn(user: UserAPIData): React$Node {
+    const showEdit: boolean =
+      CurrentUser.isLoggedIn() && CurrentUser.userID() === user.id;
+    return (
+      <React.Fragment>
+        {showEdit && this._renderEditUserButton()}
+        <div className="about-user-section">
+          <Avatar user={user} imgClass="Profile-img" />
+        </div>
+        <div className="about-user-section">
+          <h3>{user && user.first_name + " " + user.last_name}</h3>
+        </div>
+        {!_.isEmpty(user.user_links) ? (
+          <div className="about-user-section">
+            <div className="col-12">
+              <h3>Links</h3>
+              <div>{this._renderLinks(user)}</div>
             </div>
+          </div>
+        ) : null}
+      </React.Fragment>
+    );
+  }
+
+  _renderRightColumn(user: UserAPIData): React$Node {
+    return (
+      <React.Fragment>
+        {user.about_me && this._renderAboutMe(user)}
 
-            {user && !_.isEmpty(user.user_links) ? (
-              <div className="row about-user-section">
-                <div className="col-12">
-                  <h2 className="text-uppercase">Links</h2>
-                  <div>{this._renderLinks()}</div>
-                </div>
-              </div>
-            ) : null}
-
-            {user && !_.isEmpty(user.user_files) ? (
-              <div className="row about-user-section">
-                <div className="col">
-                  <h2 className="text-uppercase">Files</h2>
-                  <div>{this._renderFiles()}</div>
-                </div>
-              </div>
-            ) : null}
+        {user && !_.isEmpty(user.user_technologies)
+          ? this._renderAreasOfInterest(user)
+          : null}
+
+        {user && !_.isEmpty(user.user_files) ? (
+          <div className="about-user-section">
+            <div className="col">
+              <h2 className="text-uppercase">Files</h2>
+              <div>{this._renderFiles()}</div>
+            </div>
           </div>
-        </div>
+        ) : null}
       </React.Fragment>
     );
   }
 
-  _renderTechnologies(): ?Array<React$Node> {
-    const user: UserAPIData = this.state.user;
+  _renderAboutMe(user: UserAPIData): React$Node {
     return (
-      user &&
-      user.user_technologies && (
-        <TagsDisplay tags={user && user.user_technologies} />
-      )
+      <div className="about-user-section">
+        <div className="col-12">
```

Comment:
since you removed the `row` class here, should also remove the `col` (they need each other to work properly, and neither is needed here)